client/lib/popup.js

```diff
@@ -161,6 +161,9 @@ window.Popup = new class {
     const $element = $(element);
     return () => {
       windowResizeDep.depend();
```

Comment:
You could use the new `Utils.windowResizeDep.changed()` here and remove the `windowResizeDep` declaration.